01-path-basics/101-start-here/scripts/lab-ide-build.sh

```diff
@@ -19,10 +22,9 @@ curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s htt
 chmod +x kubectl && sudo mv kubectl /usr/local/bin/
 echo "source <(kubectl completion bash)" >> ~/.bashrc
 
-# Install kops
-curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64
-chmod +x kops-linux-amd64
-sudo mv kops-linux-amd64 /usr/local/bin/kops
+# Install Heptio Authenticator
+go get -u github.com/heptio/authenticator/cmd/heptio-authenticator-aws
```

Comment:
What is the value of `go get` instead of downloading a specific release from https://github.com/heptio/authenticator/releases?

Reply:
None. We can swap this out.

Reply:
We should pull this from our builds from our docs site - https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html#get-started-kubectl

Reply:
Future commit has both kubectl and heptio-authenticator-aws vended from EKS.  Technically you can use either, but for purposes of the workshop, let's stick with what AWS provides.